plugins/clipboard/plugin.js

```diff
@@ -197,11 +197,7 @@
 					return false;
 				}
 
-				var types = dataTransfer.getTypes(),
-					isFileOnly = types.length === 1 && types[ 0 ] === 'Files',
-					containsFile = dataTransfer.getFilesCount() === 1;
```

Comment:
Verifying if there is only a single file in dataTransfer is skipped now. I assume if we have anything in the clipboard and it is `file` type then we can take something at `0` index. And it is impossible to have more than one thing in the clipboard pasted, so this check seems to be unnecessary. Was that your way of thinking about it?

Reply:
Well, it is possible (e.g. Office always pastes HTML alongside RTF and screenshot). It's also possible to create such a clipboard data synthetically. There is also a case of drag & drop, when user can drop several files at once.

But at the moment we support pasting only one image at the time and we don't support d&d. And these two things are actually nice features. I'll create issues for them.

Back to the question: yup, as long as we support only one image, IMO we should be safe with the current condition.

Reply:
On the second thought I've readded it just in case (as we never check if there is acutally a file inside clipboard, we just assume so). Better safe than sorry.